src/js/bodyparser.js

```diff
@@ -0,0 +1,40 @@
+// Module to add body parsing to express.
+"use strict";
+var namespace  = "gpii.express.middleware.bodyparser";
```

Comment:
This header material will confuse readers. Rather than assign the namespace to a variable, you should continue to use it in full. For example, 

var gpii = fluid.registerNamespace("gpii");
fluid. registerNamespace("gpii.express.middleware.bodyparser");

gpii.express.middleware.bodyparser.json = function ....

The intent of the namespace system is not just to prevent collision on the part of implementors, but to supply clarity for readers when parsing and searching through the source code - for example, getting easy and quick answers to "find me all definitions from namespace X" etc.